topeft/modules/corrections.py

```diff
@@ -892,6 +945,75 @@ def AttachPerLeptonFR(leps, flavor, year):
     else:
         leps['fliprate'] = np.zeros_like(leps.pt)
 
+
+def AddPerPhotonFR(events,ph,year,closureTest=False):
+    # Get the fake rates lookup object
+    if year == "2016APV": year_name = "UL16APV"
+    elif year == "2016": year_name = "UL16"
+    elif year == "2017": year_name = "UL17"
+    elif year == "2018": year_name = "UL18"
+    else: raise Exception(f"Not a known year: {year}")
+
+    pt_edges = np.array([20,50,90,120])
```

Comment:
We should avoid hard coding these. Can they be loaded from the correction file or even axes.py?

Reply:
Good point. I will do that.